### PR TITLE
more than three parameters Task

### PR DESCRIPTION
**Pull request by motaz-hashhoush**

### CODE REVIEW

File: Three_parameters.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from pprint import pprint
+
+
+class Three_parameters:
```
Comment (mohamadhase) [inline Three_parameters.py:4]: please add comments to the class and function members

File: Three_parameters.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from pprint import pprint
```
Comment (mohamadhase) [inline Three_parameters.py:1]: what is this?

File: Three_parameters.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+                self.functions.append(key)
+
+    def handle_parameters(self) -> None:
+        """function that chaek number of parameters in each function
+        and print message to change parameters
+        """
+
+        for func in self.functions:
+            if ',' in func and func.count(',') > 2:
+                print('more than three parameters in this function =>', func)
```
Comment (mohamadhase) [inline Three_parameters.py:22]: edit this line make the function store the result in list or dict to print them later with other issues 
not print in line

**Review by mohamadhase — CHANGES_REQUESTED** [review]

File: Three_parameters.py (hunk 2 above)
Comment (motaz-hashhoush) [reply Three_parameters.py:1]: this is pretty print function, to print the issues beautifully and easy to understand

**Pull request merged**